core/cap-0040.md

```diff
@@ -204,6 +204,11 @@ hint. Validators would be required to compare the full signature twice for both
 signers, rather than once if the hint allows for pruning signers of the
 different type.
 
+### Empty payload is allowed
+
+This CAP allows the signer to use an empty payload. There are no issues with
+signing an empty payload, so we see no need to special case this scenario.
```

Comment:
We could be more specific as to who/what "There" is:
```suggestion
This CAP allows the signer to use an empty payload. Ed25519 specifies no issues with
signing an empty payload, so we see no need to special case this scenario.
```

Reply:
I removed this section